### PR TITLE
Derive std traits for Target.

### LINKED ISSUE #6: Implement missing imps for Target
`Target` is missing the following traits:

- [ ] `Clone`
- [ ] `Copy`
- [ ] `Eq`
- [ ] `PartialEq`
- [ ] `Ord`
- [ ] `PartialOrd`
- [ ] `Hash`

### PR DESCRIPTION
**Pull request by mjkillough**
Fixes #6.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -157,7 +157,7 @@ mod filter;
 mod filter;
 
 /// Log target, either stdout or stderr.
-#[derive(Debug)]
+#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```
Comment (KodrAus) [inline src/lib.rs]: Hmm, `Ord` and `PartialOrd` probably don't make sense for this `enum`. I think we should remove them, but the rest look good.

Comment (mjkillough) [reply src/lib.rs]: Sorry, I'd missed your comment on #6. I agree: I don't think they make sense, so I've removed them in the latest commit.

**Comment by mjkillough** [comment]
@KodrAus - Thanks a lot for the review! Hopefully the latest commit addresses your comment. Please can you take another look?

**Comment by sebasmagri** [comment]
Sure @KodrAus! Thanks!

**Review by Scutua — APPROVED** [review]

**Pull request merged**